### DISCUSSION TITLE
Détail de Reflet  sur une commande digitale

### DISCUSSION THREAD

**Discussion by StephanBerio** [witsa/synapps#789] [Q&A] [answered]
Bonjour,
un détail de reflet digital commandable sur un Marche/Arrêt donne un Marche texte blanc sur fond noir et un Arrêt texte noir sur fond blanc.
Comment changer l'aspect, exemple si en Marche avoir un texte noir sur fond vert. Merci. Stéphan

**Comment by Raphwit** [comment, marked as answer]
Hello, il faut ajouter un acteur HTML qui va récupérer à l'aide de sélecteur CSS le bouton afin de lui changer les paramètres que l'on souhaite.
Il est préférable de rajouter une classe avec un nom personnalisé sur l'acteur détails de reflet afin de ne sélectionner uniquement que cet acteur dans notre CSS.

Voici une scène qui met en pratique ces explications : 

```
SYNAPPS-STUDIO-SCENE|{"config":{"key":"scene2","name":"Scène 2"},"leadActor":{"type":"layout/stack","key":"stack1","children":[{"type":"redy/reflect-details","key":"reflect-details1","properties":{"reflectPath":"R00051.R00010","classNames":"myClass"}},{"type":"display/html","key":"html1","properties":{"content":"<style> \n\n.myClass label.btn.btn-light.toggle-on{\n  background-color: red !important;\n  color: black !important\n}\n\n.myClass label.btn.btn-light.toggle-off{\n  background-color: blue !important;\n  color: white !important;\n}\n\n</style>"}}]}}
```

**Comment by StephanBerio** [comment]
super merci beaucoup
